gazebo_msgs/srv/SpawnModel.srv

```diff
@@ -1,6 +1,6 @@
 string model_name                 # name of the model to be spawn
 string model_xml                  # this should be an urdf or gazebo xml
-string model_namespace            # spawn model and all ROS interfaces under this namespace
+string robot_namespace            # spawn robot and all ROS interfaces under this namespace
```

Comment:
Seems unnecessary to break a ROS message type mid-release - if I'm not mistaken this would change the md5 sum and is not API compatible. Can you just change the comment?